### ISSUE TITLE
Move MacVim when title bar is hidden?

### ISSUE THREAD

**Issue by nyngwang** [macvim-dev/macvim#1181]
State: Currently impossible.
Problem: When titlebar option Hidden selected, no way to move the MacVim editor.
System: MacOSX Big Sur 11.2.3 (20D91)
Picture: 

![image](https://user-images.githubusercontent.com/24765272/112825718-af0d4300-90be-11eb-9ca5-a7a1ed99ab8f.jpeg)

Is this a feature?

**Comment by ychin** [comment]
Yes this is just how it works. It's mostly useful if you already have the window at a particular place and don't want the distraction of the title bar. If you want to be able to move the window, don't hide the titlebar.

**Comment by nyngwang** [comment]
> Yes this is just how it works. It's mostly useful if you already have the window at a particular place and don't want the distraction of the title bar. If you want to be able to move the window, you may want to just leave the titlebar visible.

Do we have a toggle shortcut for this? So I won't have to do this from preference every time.

**Comment by ychin** [comment]
No we don't currently have one.

**Comment by ychin** [comment]
The main reason for this feature's existence is mostly to make it so that when you are say pinning the app to the left or the right side of the screen, you can fully utilize the screen space, but it's not really designed to be dragged around. Usually that's done by third-party tools like Spectacle which handles moving the window for you (although you can also use the Window menu). A title-less window is generally an oddity in macOS, since most macOS apps do have one, so I don't think it's worth adding too much feature support for this.

**Comment by gsa9** [comment]
In Alacritty you can move the window and resize it even when the window's title bar is hidden, which is a very nice feature to have (for me). I miss it very much in MacVim.
